### ISSUE TITLE
Start simulation on an EC2 cluster doesn't  pass the correct information to taskflow/start

### ISSUE THREAD

**Issue by cjh1** [Kitware/HPCCloud#310]
We need to pass: 

- server type
- aws profile
- cluster size

I am seeing the following:

{'cluster': {'_id': '56fda2aa0640fd748c03f491'}

Not sure what the id is referring to? It doesn't seem to be a cluster.

**Comment by TristanWright** [comment]
Let's knock this out tomorrow, I think I have close to what it should be.

**Comment by TristanWright** [comment]
fixed in #315

**Comment by cjh1** [comment]
I just seem to get the following:

```javascript
"cluster": {
  "_id": "aws profile name"  
}
```

What we need should be:

```javascript
"cluster": {
  "type": "ec2",
  "machineType": < the machine type of the nodes in the cluster >,
  "clusterSize": < the number of nodes in the cluster >,
  "profileId": < the _id of the aws profile the user selected >
}
```
